### DISCUSSION TITLE
How to integrate Teams AI with Microsoft Graph data

### DISCUSSION THREAD

**Discussion by JonoSuave** [microsoft/hack-together-teams#72] [Q&A]
In case this got missed in the conversation from today's Ask the experts session: https://github.com/microsoft/hack-together-teams/discussions/59#discussioncomment-6156307

**Comment by waldekmastykarz** [comment]
Thank you for bringing it up. It's been mentioned during the session indeed and thank you for including the pointer. I'll close this thread in favor of the link you've included.
